paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1189,10 +1230,15 @@ def on_enter_deployed(self) -> None:
         line = f"Deployment of {self.deployment_version.short_sha_repr()} for {self.deploy_group} complete"
         _log(service=self.service, component="deploy", line=line, level="event")
         self.send_manual_rollback_instructions()
+
         if self.any_slo_failing() and self.auto_rollbacks_enabled():
             self.ping_authors(
                 "Because an SLO is currently failing, we will not automatically certify. Instead, we will wait indefinitely until you click one of the buttons above."
             )
+        elif self.any_metric_failing() and self.auto_rollbacks_enabled():
+            self.ping_authors(
+                "Because a metric is currently failing, we will not automatically certify. Instead, we will wait indefinitely until you click one of the buttons above."
```

Comment:
hmm, might be nice to be a little more descriptive here - maybe something like:
```suggestion
                "Because a rollback-triggering metric for this service is currently failing, we will not automatically certify. Instead, we will wait indefinitely until you click one of the buttons above."
```
just to make it clear what type of metric we're referring to?